content/en/blog/2022/rust-instrumentation.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Rust & Otel Instrumentation
+linkTitle: Rust & Otel Instrumentation
+date: 2022-09-21
+author: '[Yoav Danieli](https://github.com/aspecto-io) (Aspecto)'
+canonical_url: https://www.aspecto.io/blog/distributed-tracing-with-opentelemetry-rust/
+---
+
+# Getting Started: OpenTelemetry in Rust
+
+![](https://lh3.googleusercontent.com/vlVY92XQhT69sNU4ZukikMN6fBpftvvf3Qs1zizIO1jJiwhB3giIpFE4ZM1q09JMdDQvl8Z7ceBFh4sebPYlrrDsdstJUILalWw3IW6OMAci2bwusf5DUtILclDYSg3zYGsUXhrlCvPzPtvmZx2UGgWj79rRFUCWDzzAvaQeBMEjlv0swZETT-XEiQ)
```

Comment:
Put the blog post into a folder and the image alongside with it, check the other blog posts for examples. Also provide an image description.